### ISSUE TITLE
Danish language

### ISSUE THREAD

**Issue by mtandrup** [yuliskov/SmartTube#1002]
Hi

I would like to request that Danish language be added to the language selection in SmartTubeNext.

**Comment by mtandrup** [comment]
How would go about doing this?

Is there a list or an index or something with all the words that need translating?

**Comment by Vistaus** [comment]
Yes, there's a strings.xml file: https://github.com/yuliskov/SmartTubeNext/blob/master/common/src/main/res/values/strings.xml

Fork the repo, clone it, create a folder _values-da_ in common/src/main/res/values/ and copy the English strings.xml file (from the link above) to the values-da folder. Then open the file and translate all of the words of the interface, i.e. Home, Search, Sign In, etc. When you're done, push the changes and open a pull request to get the translation merged.

**Comment by mtandrup** [comment]
Seems that Danish language has been added in the meantime.
